lib/archethic_web/aeweb/domain.ex

```diff
@@ -56,9 +58,10 @@ defmodule ArchethicWeb.AEWeb.Domain do
             ownerships: [ownership = %Ownership{secret: secret} | _]
           }} <- ReferenceTransaction.fetch_last(tx_address),
          {:ok, cert_pem} <- Map.fetch(json_content, "sslCertificate"),
-         %{extensions: extensions} <- EasySSL.parse_pem(cert_pem),
-         {:ok, san} <- Map.fetch(extensions, :subjectAltName),
-         ^domain <- String.split(san, ":") |> List.last(),
+         [{:Certificate, certificate_der, _}] <- :public_key.pem_decode(cert_pem),
+         %{all_domains: all_domain_names} <-
+           EasySSL.parse_der(certificate_der, all_domains: true),
```

Comment:
You could directly use `EasySSL.pem_decode/2` instead of converting the pem to der format